components/raftstore/src/store/metrics.rs

```diff
@@ -313,34 +319,39 @@ lazy_static! {
 
     /// Waterfall Metrics
     pub static ref STORE_WF_BATCH_WAIT_DURATION_HISTOGRAM: Histogram =
-        register_histogram!(
+        register_histogram_with_registry!(
             "tikv_raftstore_store_wf_batch_wait_duration_seconds",
             "Bucketed histogram of proposals' wait batch duration.",
-            exponential_buckets(0.00001, 2.0, 26).unwrap()
+            exponential_buckets(0.00001, 2.0, 26).unwrap(),
+            HIGH_PRIORITY_REGISTRY
         ).unwrap();
     pub static ref STORE_WF_SEND_TO_QUEUE_DURATION_HISTOGRAM: Histogram =
-        register_histogram!(
+        register_histogram_with_registry!(
             "tikv_raftstore_store_wf_send_to_queue_duration_seconds",
             "Bucketed histogram of proposals' send to write queue duration.",
-            exponential_buckets(0.00001, 2.0, 26).unwrap()
+            exponential_buckets(0.00001, 2.0, 26).unwrap(),
+            HIGH_PRIORITY_REGISTRY
         ).unwrap();
     pub static ref STORE_WF_BEFORE_WRITE_DURATION_HISTOGRAM: Histogram =
-        register_histogram!(
+        register_histogram_with_registry!(
             "tikv_raftstore_store_wf_before_write_duration_seconds",
             "Bucketed histogram of proposals' before write duration.",
-            exponential_buckets(0.00001, 2.0, 26).unwrap()
+            exponential_buckets(0.00001, 2.0, 26).unwrap(),
+            HIGH_PRIORITY_REGISTRY
         ).unwrap();
     pub static ref STORE_WF_WRITE_KVDB_END_DURATION_HISTOGRAM: Histogram =
-        register_histogram!(
+        register_histogram_with_registry!(
             "tikv_raftstore_store_wf_write_kvdb_end_duration_seconds",
             "Bucketed histogram of proposals' write kv db end duration.",
-            exponential_buckets(0.00001, 2.0, 26).unwrap()
+            exponential_buckets(0.00001, 2.0, 26).unwrap(),
+            HIGH_PRIORITY_REGISTRY
         ).unwrap();
     pub static ref STORE_WF_WRITE_END_DURATION_HISTOGRAM: Histogram =
-        register_histogram!(
+        register_histogram_with_registry!(
             "tikv_raftstore_store_wf_write_end_duration_seconds",
             "Bucketed histogram of proposals' write db end duration.",
-            exponential_buckets(0.00001, 2.0, 26).unwrap()
+            exponential_buckets(0.00001, 2.0, 26).unwrap(),
```

Comment:
What is the difference between "write db end duration" and "write kv db end duration"?

Reply:
"write db end duration" = "write kv db end duration" + "write raft db end duration"